code/game/machinery/machinery.dm

```diff
@@ -245,13 +245,13 @@ Class Procs:
 
 /obj/machinery/proc/max_part_rating(var/type) //returns max rating of installed part type or null on error(keep in mind that all parts have to match that raiting).
 	if(!istype(type,/obj/item/weapon/stock_parts))
-		return
+		return 1
 	var/list/obj/item/weapon/stock_parts/parts = list()
 	for(var/list/obj/item/weapon/stock_parts/P in component_parts)
 		if(istype(P, type))
 			parts.Add(P)
 	if(!parts.len)
-		return
+		return 1
```

Comment:
Why `error case` return `1` instead `null` as it was described in comment near proc head?